src/ansys/systemcoupling/core/adaptor/impl/injected_commands.py

```diff
@@ -100,19 +129,17 @@ def _wrap_add_participant(
     if input_file := kwargs.get("input_file", None):
         part_mgr.upload_file(input_file)
```

Comment:
We talked about this briefly when I was working on this, but does this indirect method of importing Session allow us to call upload_file directly from Session and remove the method from the participant manager?

Reply:
Thanks! I knew I had mentioned this and said that there was some aspect that we might be able to clean up, but I couldn't remember what it was. OK, I will see about rolling that change into this PR too.

Reply:
I have made that change now.